.circle/packagecloud.sh

```diff
@@ -120,6 +120,14 @@ function deploy() {
      continue
     fi
 
+    # NOTE: At the moment we only support staging unstable Bionic packages
+    if [ "${PKG_OS}" = "bionic" ]; then
+      if [ "${PACKAGECLOUD_REPO}" != "staging-unstable" ]; then
```

Comment:
It's not enough, since there is also related `staging-enterprise-unstable` repo

Reply:
https://github.com/StackStorm/st2-packages/pull/593